src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -424,10 +424,7 @@ private void processIter(DetailAST root, AstState astState) {
      */
     private static SortedSet<AbstractCheck> createNewCheckSortedSet() {
         return new TreeSet<>(
-                Comparator.<AbstractCheck, String>comparing(check -> check.getClass().getName())
-                        .thenComparing(AbstractCheck::getId,
-                                Comparator.nullsLast(Comparator.naturalOrder()))
-                        .thenComparing(AbstractCheck::hashCode));
+                Comparator.comparing(AbstractCheck::hashCode));
```

Comment:
I understand this code but I am not able to create any test cases for this.

Reply:
Not clear what mutation you tried to kill 
As you already have some success at https://github.com/checkstyle/checkstyle/pull/13975